src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -12,9 +12,21 @@ namespace Microsoft.ML
     public static class PermutationFeatureImportanceExtensions
     {
         /// <summary>
-        /// Permutation Feature Importance is a technique that calculates how much each feature 'matters' to the predictions.
-        /// Namely, how much the model's predictions will change if we randomly permute the values of one feature across the evaluation set.
-        /// If the quality doesn't change much, this feature is not very important. If the quality drops drastically, this was a really important feature.
+        /// Permutation feature importance (PFI) is a technique to determine the global importance features in a trained
+        /// machine learning model. PFI works by taking a labeled dataset, and then, going feature by feature, the values
+        /// for that feature are permuted, and the resulting change in the metric values for the task is computed. The
```

Comment:
`metric values for the task` is not a well-defined term. Could you add an example?
```suggestion
        /// for that feature are permuted, and the resulting change in the metric values for the task (e.g., for classification task, accuracy and area-under-the-curve are two common metrics) is computed. The
```

Reply:
Thanks! I rewrote this section.